e2e/tests/parse_params_test.go

```diff
@@ -178,8 +178,8 @@ func (s *suite) newSharedMsgUpdateParams(params paramsAnyMap) cosmostypes.Msg {
 			msgUpdateParams.Params.ApplicationUnbondingPeriodSessions = uint64(paramValue.value.(int64))
 		case sharedtypes.ParamGatewayUnbondingPeriodSessions:
 			msgUpdateParams.Params.GatewayUnbondingPeriodSessions = uint64(paramValue.value.(int64))
-		case sharedtypes.ParamComputeUnitsToTokensMultiplier:
-			msgUpdateParams.Params.ComputeUnitsToTokensMultiplier = uint64(paramValue.value.(int64))
+		case sharedtypes.ParamComputeUnitsToPpoktMultiplier:
```

Comment:
I believe we should keep CUTTM and maintain optionality of what the unit is.

Don't want to hard code `ppokt` in the name.